config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -133,7 +133,7 @@ periodics:
             - name: TEST_WINDOWS
               value: "true"
             - name: KUBERNETES_VERSION
-              value: "v1.23.5"
+              value: "v1.25.4"
```

Comment:
I don't see a 1.25.4 image yet, @mboersma @willie-yao is that one currently getting published?

`az vm image list --publisher cncf-upstream --offer capi-windows -o table --all`